TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Knative Trademark committee"
+linkTitle: "Trademark committee"
+weight: 50
+type: "docs"
+aliases:
+  - /contributing/trademark-committee/
+---
+
+The Knative Trademark Committee (KTC)
+[advises the trademark owner on the usage and enforcement of Knative logos, names, and trademarks](./GOVERNANCE.md).
+
+It is a separate body from the [Steering Committee](./STEERING-COMMITTEE.md) to
+better represent the needs of vendors and other companies which market products
+in conjuction with the Knative mark.
+
+The governance of Knative is an open, living document, and will continue to
+evolve as the community and project change.
+
+- [Charter](#charter)
+- [Delegated Authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision Process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Composition](#composition)
+  - [Maintaining the Trademark Committee](#maintaining-the-trademark-committee)
+
+## Charter
+
+1. Advise the Trademark owner on the definition and enforcement of the "Knative"
+   trademark usage and branding guidelines.
+
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+
+1. Review and approve the scope and definition of additional derivative Knative
+   marks (e.g. Knative Serving, Knative Core, etc).
+
+1. Only the Trademark Committee can redefine its own charter, membership, and
+   authority over Knative Trademark, conformance, and brand decisions.
+
+It is expected that the Trademark Committee will delegate one or more marks to
+the steering committee to provide branding for nascent efforts which are not yet
+subject to conformance and third-party licensing.
```

Comment:
I think this sentence more properly belongs and flows as the last sentence of the "Delegated Authority" section below.

Reply:
See my earlier comment; given that we spent a half-hour discussing how Steering would be able to use the mark, it seemed worth binding the KTC (via the charter) to provide such a name. If others feel that's inappropriate, I'm happy to take it out.

Reply:
Moved in the meeting today.